python/tests/aggregate_tests/test_interval_count_col.py

```diff
@@ -0,0 +1,100 @@
+from .aggtst_base import TstView
+
+
+class aggtst_interval_count_col(TstView):
+    def __init__(self):
+        # Validated on Postgres
+        self.data = [{"f_c1": 5, "f_c2": 5, "f_c3": 5, "f_c4": 5, "f_c5": 5, "f_c6": 5}]
+        self.sql = """CREATE MATERIALIZED VIEW interval_count_col AS SELECT
+                      COUNT(c1_minus_c2) AS f_c1,
```

Comment:
no need to use all columns in all views, but since you wrote them, let's leave them in.